src/Image.php

```diff
@@ -20,9 +20,7 @@ public function __construct(
         ?string $email = null,
         ?string $presetName = null,
     ) {
-        if ($email !== null) {
-            $this->setEmail($email);
-        }
```

Comment:
This was changed due to IDE complaining about constructor not calling its parent. I think its fair enough)